pysheaf.py

```diff
@@ -33,12 +34,11 @@ def __init__(self,dimension,compactClosure=True,cofaces=None, name=None):
         self.name = name
```

Comment:
I am not sure that you want the name to be the same as the ID. You might want to be able to call the cell something, but ID's tie the cell to the complex where a name may have nothing to do with the complex. So I would give the cell a unique id, assigned when it joins a complex.
```
def __init__ (self, dimension, ..., name=None, id=None)
```